src/kOS.Safe/Compilation/KS/kRISC.tpg

```diff
@@ -137,8 +138,9 @@ directive            -> lazyglobal_directive; // Add to this list later if we ma
 
 // ------------ statements --------------------
 
+empty_stmt     -> EOI;
 set_stmt       -> SET varidentifier TO expr EOI;
-if_stmt        -> IF expr instruction (ELSE instruction)? EOI?;
+if_stmt        -> IF expr instruction EOI? (ELSE instruction EOI?)?;
```

Comment:
To support the old backward compatibility where this was legal:
```
if x = 0 {
  print "true".
}. else {
  print "false".
}.
```
in which there was a dot *between* the if and the else.  This slight change has a minor effect on Compiler.cs in the VisitIfStatement() method.